packages/util/strutil.go

```diff
@@ -27,3 +27,18 @@ func TimeOrNever2(t time.Time, never string) string {
 func TimeOrNever(t time.Time) string {
 	return TimeOrNever2(t, "never")
 }
+
+type ShortStringable interface {
+	ShortString() string
+}
+
+func SliceShortString[E ShortStringable](slice []E) string {
```

Comment:
maybe we can use
```suggestion
func SliceShortString[E ShortStringable](args ...E) string {
```
instead?

Reply:
Is there any benefit? It would be easier to call if you have several separate objects but you would have to use `...` for a slice. I see no real difference.

Reply:
It was only a suggestion because in the golang standard libs its often used that way. Example:
```go
// Sprint formats using the default formats for its operands and returns the resulting string.
// Spaces are added between operands when neither is a string.
func Sprint(args ...any) string {
```
But feel free to ignore it.